mlibrary/prejudice
# May 2024 dependency updates.

# Overview
This pull request uninstalls all `eslint`-related dependencies and follows the new setup, which added these dependencies:
- `@eslint/js`
- `globals`

These dependencies have been removed:
- `eslint-config-standard`
- `eslint-plugin-import`
- `eslint-plugin-node`
- `eslint-plugin-promise`
- `eslint-webpack-plugin`

This pull request also updates:
- `@babel/cli`
- `@babel/core`
- `@babel/eslint-parser`
- `@babel/preset-env`

## Testing
- Run the tests to make sure they pass (`npm run test`).
  - Break the new/updated unit tests to make sure they're working properly.
- Build the repository (`npm run build`).
- Make sure the PR is consistent in repositories that use `prejudice`.
